eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -603,6 +624,11 @@ public void registerHealthCheck(HealthCheckHandler healthCheckHandler) {
         }
     }
 
+    @Override
+    public void registerEventListener(EurekaEventListener eventListener) {
```

Comment:
As all listeners can be injected via constructor, there is no pressing need to have this method. If we need it, we should add unregisterEventListener as well.